### PR TITLE
json feed: support post data

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Add support for POST data and headers in JSON Feed.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Review by teizenman — APPROVED** [review]

File: Packs/FeedJSON/Integrations/FeedJSON/FeedJSON_description.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ These fields also support the use of API key headers. To use API key headers, sp
 * **JMESPath Extractor** - JMESPath expression for extracting the indicators from. You can check the expression in 
 the [JMESPath site](http://jmespath.org/) to verify this expression will return the following array of objects.
 * **JSON Indicator Attribute** - JSON attribute whose value is the indicator. Default is 'indicator'.
+* **Headers** - Headers to add to the http request. Specify each header on a single line in the format: 'Name: Value'. For example:
```
Comment (amshamah419) [inline Packs/FeedJSON/Integrations/FeedJSON/FeedJSON_description.md:12]: Missing example

File: Packs/FeedJSON/Integrations/FeedJSON/README.md [hunk 2]
```diff
@@ -22,8 +22,10 @@ Use the JSON feed integration to fetch indicators from a JSON feed. This integra
...
     | Username + Password | The credentials used to access feeds that require basic authentication. These fields also support the use of API key headers. To use API key headers, specify the header name and value in the following format: `_header:<header_name>` in the **Username** field and the header value in the **Password** field. | 
-    | JMESPath Extractor | The JMESPath expression for extracting the indicators from. You can check the expression in the [JMESPath site](http://jmespath.org/) to verify this expression will return the following array of objects. |
+    | JMESPath Extractor | The JMESPath expression for extracting the indicators from. You can check the expression in the [JMESPath site](http://jmespath.org/) to verify this expression will return the following array of objects. |    
     | JSON Indicator Attribute | The JSON attribute whose value is the indicator. The default is "indicator". |
+    | POST Data | Send specified data in a POST request. When specified, by default will add the header: `Content-Type: application/x-www-form-urlencoded`. To specify a different Content-Type (for example: application/json) use the **Header** config param. |
```
Comment (amshamah419) [inline Packs/FeedJSON/Integrations/FeedJSON/README.md:27]:
```suggestion
    | POST Data | Send specified data in a POST request. When specified, by default will add the header: `Content-Type: application/x-www-form-urlencoded`. To specify a different Content-Type (for example: application/json) use the **Headers** config param. | 
```

File: Packs/FeedJSON/Integrations/FeedJSON/FeedJSON_description.md [hunk 3]
```diff
@@ -9,6 +9,12 @@ These fields also support the use of API key headers. To use API key headers, sp
 * **JMESPath Extractor** - JMESPath expression for extracting the indicators from. You can check the expression in 
 the [JMESPath site](http://jmespath.org/) to verify this expression will return the following array of objects.
 * **JSON Indicator Attribute** - JSON attribute whose value is the indicator. Default is 'indicator'.
+* **Headers** - Headers to add to the http request. Specify each header on a single line in the format: 'Name: Value'. For example: 
+* **POST Data** - Send specified data in a POST request. When specified, by default will add the header: 'Content-Type: application/x-www-form-urlencoded'. To specify a different Content-Type (for example: application/json) use the **Header** config param.
```
Comment (amshamah419) [inline Packs/FeedJSON/Integrations/FeedJSON/FeedJSON_description.md:13]:
```suggestion
* **POST Data** - Send specified data in a POST request. When specified, by default will add the header: 'Content-Type: application/x-www-form-urlencoded'. To specify a different Content-Type (for example: application/json) use the **Headers** config param.
```

File: Packs/FeedJSON/Integrations/FeedJSON/FeedJSON.yml [hunk 4]
```diff
@@ -94,6 +96,20 @@ configuration:
   name: indicator
   required: false
   type: 0
+- additionalinfo: "Send specified data in a POST request. When specified, by default\
+    \ will add the header: 'Content-Type: application/x-www-form-urlencoded'. To specify\
+    \ a different Content-Type (for example: application/json) use the Header config\
```
Comment (amshamah419) [inline Packs/FeedJSON/Integrations/FeedJSON/FeedJSON.yml:101]:
```suggestion
    \ a different Content-Type (for example: application/json) use the Headers config\
```

File: Packs/FeedJSON/Integrations/FeedJSON/FeedJSON_description.md (hunk 1 above)
Comment (glicht) [reply Packs/FeedJSON/Integrations/FeedJSON/FeedJSON_description.md:12]: good catch. will add.

**Pull request merged**